pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -343,7 +349,7 @@ def _updates_infectiousness(self, person: Person, time: float):
                                            InfectionStatus.Dead,
                                            InfectionStatus.Vaccinated]:
                 person.infectiousness = 0
-                person.infection_start_time = None
+                # person.infection_start_time = None
```

Comment:
Try to avoid commented code on main - if we don't need this it can be removed

Reply:
Sure, done